module/VuFind/config/module.config.php

```diff
@@ -139,6 +139,39 @@
                     ],
                 ],
             ],
+            'admin' => [
+                'type' => 'Laminas\Router\Http\Literal',
+                'options' => [
+                    'route'    => '/Admin',
```

Comment:
Is there a reason you added these routes here instead of as part of the VuFindAdmin module? Historically we have kept all of our administrative code isolated in the VuFindAdmin module so that it's very easy to completely disable it (or even delete it) if you have an environment where admin code is on a firewalled server and public VuFind is in a less secure environment.

Reply:
No good reason - I moved the configuration to the VuFindAdmin module.